app/build.gradle

```diff
@@ -257,11 +249,6 @@ android {
         exclude 'META-INF/NOTICE.txt'
     }
 
-    dexOptions {
```

Comment:
Any comments on why is this removed ?

Reply:
Ohh, sorry for not adding info on the commit. https://issuetracker.google.com/issues/183295952 this is the related issue tracker which contains more details on the issue. 
```
DSL element 'dexOptions' is obsolete and should be removed.
It will be removed in version 8.0 of the Android Gradle plugin.
Using it has no effect, and the AndroidGradle plugin optimizes dexing automatically.
```